### DISCUSSION TITLE
静态资源增添一个公益CDN

### DISCUSSION THREAD

**Discussion by wotime** [AlistGo/alist#2976] [Ideas]
找到一个新的公益CDN，使用体验感觉还不错，可以考虑加到Alist的文档里面，做备选使用。
地址如下： https://cdn.onmicrosoft.cn/

![1673269495328.png](https://bj.bcebos.com/baidu-rmb-video-cover-1/14308e61f1ef90d018b9e73e3218d895.png)

**Comment by NovaShen0724** [comment]
github的也挺快的了啊
